src/radiation/radiation.cpp

```diff
@@ -240,9 +237,14 @@ void ablate::radiation::Radiation::Initialize(solver::Range cellRangeIn) {
              * Assemble a vector of vectors etc associated with each cell index, angular coordinate, and space step?
              * The boundary has been reached if any of these conditions don't hold
              * */
+
```

Comment:
Would you mind removing the debug code?

Reply:
Ah, I didn't realize that was still in there. I can do that.